### ISSUE TITLE
!help

### ISSUE THREAD

**Issue by itsrachelfish** [itsrachelfish/fishy-bot#36]
Fishy needs a !help command. There should be some predefined variable that modules can write strings to which would be output by a help module.

**Comment by frowardd** [comment]
yeah this needs documentation it's really embarrasing, i can't even spam rainbows properly anymore

**Comment by frowardd** [comment]
![Screenshot](https://wiki.wetfish.net/upload/55c80793-2155-f8de-bebd-c73af4915ce6.png)
also the output is kinda funky, this is on Hexchat, monospace 12

**Comment by frowardd** [comment]
I don't know both rainbow commands. Sorry!
